packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/utils/helpers.dart

```diff
@@ -13,33 +10,3 @@ Future safeRun(Function action) async {
     log('Safe Error: $e');
   }
 }
-
-/// Shows a dialog that adjusts to the target operating system
-Future<T?> showAdaptiveDialog<T>({
```

Comment:
By removing this you are breaking the application's existing functionality in `lib/feature_notifications/views/notifications_page.dart`. 
I believe that this function can be replaced with Flutter's 3.13 new `showAdaptiveDialog`

Reply:
https://github.com/flutter/flutter/pull/124336